docs/feature/search/fts/tutorial.md

```
@@ -26,6 +29,12 @@ To begin, let's create the schema for this dataset.
 
 ## Creating the Table
 
+:::{div}
+The fastest and easiest way to get started with CrateDB is by deploying a
+free tier (CRFREE) cluster on [CrateDB Cloud][CrateDB Cloud Console]. Otherwise,
```

markdownlint-cli2 (0.18.1)

Reference links and images should use a label that is defined
Missing link or image reference definition: "cratedb cloud console"

(MD052, reference-links-images)